src/exports.jl

```diff
@@ -70,6 +70,8 @@ export Edge
 export EffectiveCartierDivisor
 export EllipticSurface
 export EmptyScheme
+export EnriquesBorcherdsCtx
+export Enriques_surface_automorphism_group
```

Comment:
This does not seem to fit Oscar naming guideline for non-capitalized functions' names. What about `enriques_surface_automorphism_group` ? Similar to `borcherds_...`